crates/torii/core/src/query_queue.rs

```diff
@@ -15,7 +15,7 @@ pub enum Argument {
 #[derive(Debug, Clone)]
 pub struct QueryQueue {
     pool: Pool<Sqlite>,
-    queue: VecDeque<(String, Vec<Argument>)>,
+    pub queue: VecDeque<(String, Vec<Argument>)>,
```

Comment:
Is there a reason as to why we make this public?

Reply:
ah my bad i added it will testing out things